azkaban-web-server/src/main/java/azkaban/webapp/servlet/AbstractAzkabanServlet.java

```diff
@@ -375,6 +375,21 @@ protected void writeJSON(final HttpServletResponse resp, final Object obj, final
     JSONUtils.toJSON(obj, resp.getOutputStream(), true);
   }
 
+  protected void sendErrorResponse (HttpServletResponse httpServletResponse, int status,
```

Comment:
Makes sense to have these methods in the abstract class and they will be quite useful. Thanks.